_observing-your-data/metricsanalytics.md

```diff
@@ -0,0 +1,205 @@
+
+<img src="{{site.url}}{{site.baseurl}}/images/metrics/otel-metrics.png" alt="OTel metrics dashboard" width="700"/>
+
+---
+
+## Visualizing metrics in remote clusters
+Introduced 2.14
+{: .label .label-purple }
+
+You can view metrics from remote OpenSearch clusters by using the **Metrics** tool. Select the database icon on the upper-right toolbar and choose a cluster from the **DATA SOURCES** dropdown menu, as shown in the following image. You can switch from a local cluster to a remote cluster.
+
+<img src="{{site.url}}{{site.baseurl}}/images/metrics/remote-cluster-selection.png" alt="Switching clusters using the Metrics analytics tool" width="700"/>
+
+You can also view metric visualizations from other sources alongside local metric visualizations. From the **DATA SOURCES** dropdown menu, choose the remote metric visualization to add it to the group of visualizations already shown on the dashboard. An example dashboard is shown in the following image.
+
+<img src="{site.url}}{{site.baseurl}}/images/metrics/otel-metrics-remote-cluster-selection.png" alt="Metrics dashboard" width="700"/>
+
+To learn about multi-cluster support for data sources, see [Enable OpenSearch Dashboards to support multiple OpenSearch clusters](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1388).
+
+## Creating visualizations based on custom metrics
+
+You can create visualizations using the metric data collected by your OpenSearch cluster, including Prometheus metrics and custom metrics.
+
+To create these visualizations, follow these steps:
+
+1. From the OpenSearch Dashboards main menu, navigate to **Observability** > **Metrics** > **Available Metrics**.
+2. Choose the metrics to add to your visualization and then select **Save**.
+3. When prompted for a **Custom operational dashboards/application**, choose one of the listed options. You can edit the predefined name values in the **Metric Name** field.
+4. Select **Save** to save your visualization. An example visualization is shown in the following image.
+
+<img src="{{site.url}}{{site.baseurl}}/images/metrics/metrics2.png" alt="Metrics analytics dashboard with visualizations" width="700"/>
+
+## Defining PPL queries for Prometheus metrics
+
+You can define [Piped Processing Language (PPL)]({{site.url}}{{site.baseurl}}/search-plugins/sql/ppl/index/) queries to interact with metrics collected by Prometheus. The following is an example PPL query for a Prometheus metric:
+
+```
+source = my_prometheus.prometheus_http_requests_total | stats avg(@value) by span(@timestamp,15s), handler, code
+```
+{% include copy-curl.html %}
+
+### Creating a custom visualization based on the PPL query
+
+To create a custom visualization based on the PPL query, follow these steps:
+
+1. In the **Event Analytics** window, enter your PPL query and select **Refresh**. This will display the **Explorer** page.
```

Comment:
```suggestion
1. From the **Logs** > **Event Explorer** page, enter your PPL query and select **Refresh**. 
```

Reply:
@YANG-DB I don't see an "Events analytics" page in OpenSearch Dashboards. Are we referring to the the following? 
![Uploading Screenshot 2024-06-21 at 9.39.44 AM.png…]()